components/core/src/package/archive.rs

```diff
@@ -411,7 +410,7 @@ impl PackageArchive {
 
         match self.read_metadata(file) {
             Ok(Some(body)) => {
-                let ids: Vec<String> = body.lines().map(|d| d.to_string()).collect();
+                let ids: Vec<String> = body.lines().map(ToString::to_string).collect();
```

Comment:
I think
```suggestion
                let ids: Vec<String> = body.lines().map(str::to_string).collect();
```
or
```suggestion
                let ids: Vec<String> = body.lines().map(str::to_owned).collect();
```
are more illustrative. Unfortunately clippy makes the suggestion which is most broadly applicable, but when our type is know beyond just the trait bound, I find it makes it clearer what's going on.